livedata/src/main/java/io/getstream/chat/android/livedata/entity/ChannelEntity.kt

```diff
@@ -33,7 +33,7 @@ internal data class ChannelEntity(var type: String, var channelId: String) {
     var frozen: Boolean = false
 
     /** if the channel is hidden (new messages will cause to reappear) */
-    var hidden: Boolean = false
+    var hidden: Boolean? = null
```

Comment:
Why do we need it as nullable property?

Reply:
otherwise the difference between null and false triggers the channel diff callback to believe something has changed.